### PR TITLE
Allow multi-input model export

### PR DESCRIPTION
**Pull request by wpmed92**
Add support for exporting models that have `forward` or `__call__` with multiple Tensor inputs

### CODE REVIEW

File: extra/export_model.py [hunk 1]
```diff
@@ -125,24 +127,25 @@ def export_model_webgpu(functions, statements, bufs, bufs_to_save, weight_names)
 }}
   """ + f"\n\nconst loadNet = async (device) => {{ return await fetch('net.safetensors').then(x => x.arrayBuffer()).then(x => setupNet(device, new Uint8Array(x))); }}"
 
-def export_model(model, input:Tensor, target:str):
+def export_model(model, inputs:List[Tensor], target:str):
```
Comment (dc-dc-dc) [inline extra/export_model.py:130]: switch the ordering of `inputs` and `target` to make the api cleaner with a variable number arg list

**Comment by wpmed92** [comment]
compile_efficientnet is in the CI so that tests the single input aspect, but for the multi input I'll add a unit test

File: extra/export_model.py (hunk 1 above)
Comment (wpmed92) [reply extra/export_model.py]: just fixed this

**Comment by wpmed92** [comment]
I added a unit test for model export.

**Pull request merged**